### DISCUSSION TITLE
Feature Request: Export Lists (csv, HTML format)

### DISCUSSION THREAD

**Discussion by thedaviddias** [community/community#63949] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I love the list feature but it would be nice if we had a feature that would allow us to export the list of the starred repos.

Formats like CSV or a simple HTML page would be nice. Of course, would even be better if we could access it through an API but in the meantime, I think this would be a good alternative.

**Comment by barbabbott** [comment]
_p_

**Comment by 4ff0rdsUn1c0rns** [comment]
> ... access it through an API but in the meantime, I think this would be a good alternative.

This is available.

EDIT:
https://github.com/community/community/discussions/8293#discussioncomment-4088677

... Where did I just look other than here ... Ugh LoL; something mentioned API with a link.

EDIT-2:
Here:
https://github.com/community/community/discussions/65045#discussioncomment-6831223

EDIT-3:
. Unsure if tagging in response is necessary, pending settings @thedaviddias . Apologies if not.

**Reply by omidbarzegaran85** [reply]
#####

**Reply by omidbarzegaran85** [reply]
@4ff0rdsUn1c0rns

**Reply by 4ff0rdsUn1c0rns** [reply]
> @4ff0rdsUn1c0rns 

Thought I saw your email in a repository; looked again later, failed to find it...
So; public response...

@omidbarzegaran85 ?

Yes?
